Makefile

```diff
@@ -379,6 +394,16 @@ docker-push:
 		docker push $(DOCKER_NAMESPACE)/jaeger-$$component ; \
 	done
 
+.PHONY: dockerx-push
+dockerx-push: .ensure-docker-buildx .ensure-buildx-builder
+	@while [ -z "$$CONFIRM" ]; do \
```

Comment:
Would this work in the CI as well? What's the worst that can happen if I call `make dockerx-push` but don't really mean to? I can just cancel it, can't I?

Reply:
Just copy from `make docker-push`. 
we could remove it